tests/test_grid3d/test_grid_property.py

```diff
@@ -697,3 +696,48 @@ def test_gridprop_export_bgrdecl_double(tmp_path, grid):
 
     with open(tmp_path / "actnum.DATA", "rb") as fh:
         assert b"DOUB" in fh.read()
+
+
+@pytest.mark.parametrize(
+    "simcase, props, expected_avg, description",
+    [
+        (
+            "E100_BO",
+            ["SOIL", "PRESSURE"],
+            [0.0119888, 270.03588],
+            "E100 blackoil",
+        ),
+        (
+            "E300_BO",
+            ["SOIL", "PRESSURE"],
+            [0.0119888, 270.03588],
+            "E300 blackoil",
+        ),
+        (
+            "E300_COMP",
+            ["SOIL", "PRESSURE"],
+            [0.00945095, 400.720811],
+            "E300 composite",
+        ),
+    ],
+)
+def test_simpleb8_case(simcase, props, expected_avg, description):
+    """Test a very small case that is actually ran through E100, E300, IX."""
+
+    folder = TPATH / "3dgrids" / "simpleb8"
+
+    grd = xtgeo.grid_from_file(folder / str(simcase + ".EGRID"), fformat="egrid")
+
+    assert grd.dimensions == (4, 2, 3)
+
+    for inum, prop in enumerate(props):
+        logger.info("Testing property %s for %s", prop, description)
+        gprop = xtgeo.gridproperty_from_file(
+            folder / str(simcase + ".UNRST"),
+            name=prop,
+            fformat="unrst",
+            date="20220101",
+            grid=grd,
+        )
+
+        assert gprop.values.mean() == pytest.approx(expected_avg[inum], rel=0.001)
```

Comment:
This looks a bit like a snapshot test, so maybe go all the way and make that explicit?

```suggestion
        snapshot.assert_match(str(gprop.values), f"{prop}_{simcase}")
```

That eliminates the need to parameterize over expected_avg.

Reply:
Will test 👍

Reply:
I am struggling making snapshot work; propose leave it for now?

Reply:
I propose to keep as is, since there are also some disadvantages with snapshot (potential large commits, numerical inaccuracy and perhaps formatting [since text is compared, not numerics]). Instead I add "stddev" comparison.

Reply:
I dont see those as large drawbacks, having files added to the commit does of course increase the diff, but it does not make the reviwing any more difficult as the additional lines are not logic, they are just results. It also does not increase the size of the package, only the test folder.

Like any test data it can come out of sync if there are breaking changes, but that is not very frequent in `numpy`, also, updating is as easy as running `pytest --snapshot-update`.

We are running quite a few snapshot tests already, and numerical inaccuracy has not been a problem, so dont think it will be here. Rounding the result could be done to decrease the likelyhood.